sechub-cli/src/mercedes-benz.com/sechub/cli/config_test.go

```diff
@@ -95,3 +95,41 @@ func Example_willTrailingSlashBeRemovedFromUrl() {
 	fmt.Println(config.server)
 	// Output: https://test.example.org
 }
+
+func TestValidateRequestedReportFormat(t *testing.T) {
+	// PREPARE
+	config := NewConfigByFlags()
+	config.reportFormat = "written-on-paper"
+	// EXECUTE
+	validateRequestedReportFormat(config)
+	// TEST
+	if config.reportFormat != "json" {
+		t.Errorf("Reportformat was not changed to 'json'. Got '%s'.", config.reportFormat)
+	}
+}
+
+func Example_validateRequestedReportFormatMakesLowercase1() {
+	// PREPARE
+	config := NewConfigByFlags()
+	config.reportFormat = "HTML"
+	// EXECUTE
+	validateRequestedReportFormat(config)
+	// TEST
+	fmt.Println(config.reportFormat)
+	// Output:
+	// WARNING: Given report format did contain uppercase characters which are not accepted by SecHub server - so changed to lowercase before sending
```

Comment:
```suggestion
	// WARNING: Converted report format to lowercase. Because given report format contained uppercase characters, which are not accepted by SecHub server.
```

In addition, it might be enough to provide a `NOTICE` or `HINT` rather than a `WARNING`.

Reply:
Good idea. :1st_place_medal: 
Now a NOTICE appears when converting to lower case. See latest commit.